lib/logger.js

```diff
@@ -14,15 +14,16 @@ var ut = require('util');
  */
 
 function viewed(str) {
-  var re = /\u001b\[\d+m/gm;
+  const re = /\u001b\[\d+m/gm;
```

Comment:
Move the regex to a constant in the module scope, seeing as how its used multiple times.

```
const ANSI_ESCAPE_PATTERN = /\u001b\[\d+m/gm;
```